libs/ui-watt/src/lib/components/toast/watt-toast.module.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ModuleWithProviders,
+  NgModule,
+  Optional,
+  SkipSelf,
+} from '@angular/core';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { WattToastComponent } from './watt-toast.component';
+
+@NgModule({
+  imports: [MatSnackBarModule, WattToastComponent],
+  exports: [WattToastComponent],
+})
+export class WattToastRootModule {
+  constructor(
+    @Optional()
+    @SkipSelf()
+    maybeNgModuleFromParentInjector?: WattToastRootModule
+  ) {
+    if (maybeNgModuleFromParentInjector) {
+      throw new Error(
+        'WattDanishDatetimeRootModule.forRoot registered in multiple injectors. Only call it from the core feature shell module or in the Angular testing module.'
+      );
+    }
+  }
+}
+
+/**
+ * Do not import directly. Use `WattToastModule.forRoot`.
+ */
+@NgModule()
+export class WattToastModule {
```

Comment:
suggestion: let's import the module in `DhCoreShellModule` so other developers can directly start using the toast service.